Bootloader/Src/firmware_update.c

```diff
@@ -61,6 +62,7 @@
 #define HASH_SIZE       (32U)           //!< Hashed board id value size in bytes
 #define CRC_SIZE        (4U)            //!< CRC size in bytes (CRC32)
 #define XOR_CRC_VALUE   (0xFFFFFFFFU)   //!< XOR CRC value
+#define TX_BUFFER_SIZE  (1000U)

```

Comment:
We can add comment in case we decide to use doxygen in the future.

//!< TX buffer maximum size